recipes/reindent/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "Reindent" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
```

Comment:
Would you mind following the hash jinjia2 variables from the example? (It may be helpful to have these standardized, and I don't expect them to change much)